### PR TITLE
Litecoin support 202003

### PR DESCRIPTION
**Pull request by jasonbphillips**
Brings litecoin support current to 2020-03.

---
This change is [Reviewable](https://reviewable.io/reviews/elliotchance/independentreserve/18)

### CODE REVIEW

File: tests/unit/IndependentReserve/CurrencyTest.php [hunk 1]
```diff
@@ -6,39 +6,108 @@
...
 
+    public function testXrp()
+    {
+        $this->assert(Currency::XRP, equals, 'Xrp');
+    }
+
+    public function testXrp()
+    {
+        $this->assert(Currency::XRP, equals, 'Xrp');
+    }
```
Comment (elliotchance) [inline tests/unit/IndependentReserve/CurrencyTest.php]: There are two definitions of `testXrp`

File: tests/unit/IndependentReserve/CurrencyTest.php [hunk 2]
```diff
@@ -6,39 +6,108 @@
...
+
+    public function testUsdt()
+    {
+        $this->assert(Currency::USDT, equals, 'Usdt');
+    }
+
+    public function testUsdt()
+    {
+        $this->assert(Currency::USDT, equals, 'Usdt');
+    }
```
Comment (elliotchance) [inline tests/unit/IndependentReserve/CurrencyTest.php]: There are two definitions of `testUsdt`.

**Review by elliotchance — CHANGES_REQUESTED** [review]

File: tests/unit/IndependentReserve/CurrencyTest.php (hunk 1 above)
Comment (jasonbphillips) [reply tests/unit/IndependentReserve/CurrencyTest.php]: My apologies - late night copy and paste error...

File: tests/unit/IndependentReserve/CurrencyTest.php (hunk 2 above)
Comment (jasonbphillips) [reply tests/unit/IndependentReserve/CurrencyTest.php]: My apologies - late night copy and paste error...

**Review by elliotchance — APPROVED** [review]

**Pull request merged**